### PR TITLE
Majella add networks comparison

### PR DESCRIPTION
**Pull request by MajellaH**
Fixes Issue #628
New web page with comparisons between Polkadot and Kusama networks

### CODE REVIEW

**Comment by Swader** [comment]
@w3fbot prettify please

@MajellaH I would recommend linking to Kusama's resources:

- https://guide.kusama.network for the Kusama wiki
- https://kusama.network for the landing page
- https://kusama.dotapps.io for the Polkadot-JS Apps specific to Kusama

**Comment by MajellaH** [comment]
@Swader Links added

**Comment by Swader** [comment]
If you click on View File, you can preview how your links look if you're only editing online (in Github's UI) rather than locally on your machine. You will notice the links are rendered in order one after the other, and not as a list, one under the other. To get a list, you put an asterisk or a dash in front of every new line, like so:

```md
- one
- two
- three
```

Also, you should never omit the protocol in links (`https://`) because you never know how a link is set up server-side, it might not be equipped to auto-redirect from `http` (default when omitted) to `https` (desired). Last but not least, you should always test your links (the View File function will help) - you'll notice that the last one, dotapps, does not work. Instead, it redirects to Polkadot's dotapps, not Kusama's. For best results, use the links exactly as I gave them in the comments, i.e. do not change the URLs.

File: docs/learn-kusama-vs-polkadot.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+Kusama was released as an early version of the same code to be used in Polkadot, which means they share the same underlying architecture: a multichain, heterogeneously-sharded design based on nominated proof of stake (NPoS).
+Both networks also share key innovations like on-chain governance, hot-swappable runtimes for forkless, on-chain upgrades, and cross-chain message passing (XCMP) for interoperability.
+Governance on both Polkadot and Kusama is designed to be decentralized and permissionless, giving a say in how the network is run to everyone who owns the native token (DOT for Polkadot, KSM for Kusama). Therefore, **over time the networks will evolve independently, converging or diverging according to the decisions of their respective communities.**
+
+## Key differences
+
+There are a few important distinctions to be made.
+
+![polkadot_vs_kusama](/img/polkadot_vs_kusama.png)
```
Comment (krichard410) [inline docs/learn-kusama-vs-polkadot.md]: I would recommend here to place the image inside the `assets` folder. Logan's new img optimizer will be optimizing the images inside that folder.

File: docs/learn-kusama-vs-polkadot.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+---
+
+# What is the difference between Polkadot and Kusama?
+
+Although they are built on nearly the same codebase, Polkadot and Kusama are independent, standalone networks with different priorities.  
+Kusama is wild and fast; great for bold experimentation and early-stage deployment. Polkadot is more conservative, prioritizing stability and dependability.
+
+## What the two networks have in common
+
+Kusama was released as an early version of the same code to be used in Polkadot, which means they share the same underlying architecture: a multichain, heterogeneously-sharded design based on nominated proof of stake (NPoS).
```
Comment (krichard410) [inline docs/learn-kusama-vs-polkadot.md]:
```suggestion
Kusama was released as an early version of the same code to be used in Polkadot, which means they share the same underlying architecture: a multichain, heterogeneously-sharded design based on Nominated Proof of Stake (NPoS).
```
just a nitpick =)

File: docs/learn-kusama-vs-polkadot.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+Both networks also share key innovations like on-chain governance, hot-swappable runtimes for forkless, on-chain upgrades, and cross-chain message passing (XCMP) for interoperability.
```
Comment (krichard410) [inline docs/learn-kusama-vs-polkadot.md]:
```suggestion
Both networks also share key innovations like on-chain governance, hot-swappable runtimes for forkless, on-chain upgrades, and Cross-Chain Message Passing (XCMP) for interoperability.
```

**Comment by Swader** [comment]
@w3fbot prettify

**Comment by Swader** [comment]
@MajellaH why did you close this?
@lsaether what's up with the bot, it won't prettify? Is this because of #944?

**Comment by MajellaH** [comment]
Sorry - closed in error.

**Comment by lsaether** [comment]
@Swader No I think it's just glitching on this branch since the `check-format` CI is not running.

File: docs/learn-kusama-vs-polkadot.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
+---
+id: learn-kusama-vs-polkadot
+title: Comparing Polkadot and Kusama
+sidebar_label: Comparing Polkadot and Kusama
+---
+
+# What is the difference between Polkadot and Kusama?
```
Comment (lsaether) [inline docs/learn-kusama-vs-polkadot.md]: This will cause the page to have two header titles since the title is taken from the metadata above. So it would end up rendering like the below. I would recommend changing the `title` field in the metadata and deleting the first header.

# Comparing Polkadot and Kusama

# What is the difference between Polkadot and Kusama?

Although they are built...

File: docs/learn-kusama-vs-polkadot.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+Although they are built on nearly the same codebase, Polkadot and Kusama are independent, standalone networks with different priorities.
```
Comment (lsaether) [inline docs/learn-kusama-vs-polkadot.md]: Technically they are built from the _same_ codebase so it may be better to rephrase this as saying they "share many parts of their code".

File: docs/learn-kusama-vs-polkadot.md (hunk 4 above)
Comment (MajellaH) [reply docs/learn-kusama-vs-polkadot.md]: Changed

File: docs/learn-kusama-vs-polkadot.md (hunk 5 above)
Comment (MajellaH) [reply docs/learn-kusama-vs-polkadot.md]: Changed

File: docs/learn-kusama-vs-polkadot.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+Ultimately, Kusama and Polkadot will live on as independent, standalone networks with their own communities, their own governance, and their own complementary use cases, though they will continue to maintain a close relationship, with many teams likely deploying applications to both networks.
+In the future, we’re also likely to see Kusama bridged to Polkadot for cross-network interoperability.
+Web3 Foundation and Parity Technologies remain committed to both networks going forward, providing crucial support and guidance to teams building for the ecosystem.
+
+## Explore more
+
+- [About Kusama](https://kusama.network)
+- [The Kusama Wiki](https://guide.kusama.network)
+- [Kusama on polkadot.js](https://kusama.dotapps.io)
```
Comment (krichard410) [inline docs/learn-kusama-vs-polkadot.md]:
```suggestion
- [Kusama on Polkadot-JS Apps](https://kusama.dotapps.io)
```
I'd just change the name of the Polkadot UI here

File: docs/learn-kusama-vs-polkadot.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+For those who want to validate on Kusama *and* Polkadot, the same setup and infrastructure can be used for both networks.
+
+### Use cases
+
+Polkadot is and always will be the primary network for the deployment of enterprise-level applications and those that entail high-value transactions requiring bank-level security and stability.
+The initial use case for Kusama is as a pre-production environment, a “canary network”. Building on Kusama first allows teams to test things out in a live, fully decentralized and community-controlled network with real-world conditions and lower stakes in the event of problems or bugs than on Polkadot.
+Many projects will maintain parachains on both networks, experimenting and testing new technologies and features on Kusama before deploying them to Polkadot.
+Some teams will decide just to stay on Kusama, which is likely to be a place where we see some exciting experimentation with new technologies going forward.
+Projects that require high-throughput but don’t necessarily require bank-like security, such as some gaming, social networking, and content distribution applications, are particularly good candidates for this use-case.
```
Comment (laboon) [inline docs/learn-kusama-vs-polkadot.md]:
```suggestion
Projects that require high-throughput but don’t necessarily require bank-like security, such as some gaming, social networking, and content distribution applications, are particularly good candidates for this use case.
```

File: docs/learn-kusama-vs-polkadot.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+Kusama may also prove to be the perfect environment for ambitious experiments with new ideas and new innovations in areas like governance, incentives, monetary policy, and DAOs (decentralized autonomous organizations).
+Future upgrades to the Polkadot runtime will also likely be deployed to Kusama before Polkadot mainnet. This way, not only will we be able to see how these new technologies and features will perform under real-world conditions before bringing them to Polkadot, but teams who have deployed to both networks will also get an advanced look at how their own technology will perform under those upgrades.
```
Comment (laboon) [inline docs/learn-kusama-vs-polkadot.md]:
```suggestion
Future upgrades to the Polkadot runtime will also likely be deployed to Kusama before Polkadot mainnet. This way, not only will we be able to see how these new technologies and features will perform under real-world conditions before bringing them to Polkadot, but teams who have deployed to both networks will also get an advance look at how their own technology will perform under those upgrades.
```

File: docs/learn-kusama-vs-polkadot.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+
+## Going forward
+
+Ultimately, Kusama and Polkadot will live on as independent, standalone networks with their own communities, their own governance, and their own complementary use cases, though they will continue to maintain a close relationship, with many teams likely deploying applications to both networks.
+In the future, we’re also likely to see Kusama bridged to Polkadot for cross-network interoperability.
+Web3 Foundation and Parity Technologies remain committed to both networks going forward, providing crucial support and guidance to teams building for the ecosystem.
```
Comment (laboon) [inline docs/learn-kusama-vs-polkadot.md]:
```suggestion
Web3 Foundation remains committed to both networks going forward, providing crucial support and guidance to teams building for the ecosystem.
```

**Review by laboon — CHANGES_REQUESTED** [review]

**Review by lsaether — APPROVED** [review]
